src/tutorials/exbinfile.c

```diff
@@ -40,6 +43,9 @@ int main(int argc,char **argv)
   ierr = PetscOptionsGetString(NULL,NULL,"-f",training_file,sizeof(training_file),NULL);CHKERRQ(ierr);
   ierr = PetscOptionsGetString(NULL,NULL,"-f_test",test_file,sizeof(test_file),&test_file_set);CHKERRQ(ierr);
 
+  ierr = PetscOptionsGetString(NULL,NULL,"-f_training_predictions",training_result_file,sizeof(training_result_file),&training_result_file_set);CHKERRQ(ierr);
+  ierr = PetscOptionsGetString(NULL,NULL,"-f_test_predictions",test_result_file,sizeof(test_result_file),&test_result_file_set);CHKERRQ(ierr);
+
```

Comment:
I'd again output the results to stdout. The point of the tutorial isn't to show how to make h5/binary viewers. I.e. just call ```SVMViewT*Predictions(svm,NULL);```

Reply:
Since the showcase of outputting results into stdout is implemented in ex3. I've removed it from exbinfile.